README.md

```diff
@@ -5,757 +5,886 @@
 
 # Microstates
 
-* [Intro](#intro)
-* [microstate constructor](#microstate-constructor)
-* [Composition](#composition)
-* [Static values](#static-values)
-* [Computed Properties](#computed-properties)
+Composable State Primitives with a GraphQL inspired API.
```

Comment:
I don’t think we should mention GraphQL here. This makes it feel way more heavyweight than it actually is. Composable State Primitives for JavaScript. I almost feel like we should lead with 

> Microstates is a functional runtime type system designed to ease state management in component based applications. It allows you to declaratively compose application state from atomic state machines.

How about:
> By combining lazy execution, algebraic data types and structural sharing, we created a tiny API to let you declaratively compose application state from immutable state machines.

Although, I feel though if we're going to mention state machines, then we should show a quick example of how they map together, because I don't think people (event David K. Piano) understand how they correlate.

Reply:
Agree, but I need some advice on how to express it.